### PR TITLE
Add unstyled search bar

### PR DESCRIPTION
**Pull request by Magnus-G**

### CODE REVIEW

File: source/unstyled/_search_bar.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="search-bar">
+  <form role="search">
```
Comment (tysongach) [inline source/unstyled/_search_bar.html.erb]: Nice use of `role`!

File: source/unstyled/_search_bar.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <input type="search" placeholder="Enter Search" />
```
Comment (tysongach) [inline source/unstyled/_search_bar.html.erb]: No need for ending `/` » `<input type="search" placeholder="Enter Search">`

File: source/unstyled/_search_bar.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    <button type="submit">
+      <img src="https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/search-icon-black.png" alt="Search Icon">
```
Comment (tysongach) [inline source/unstyled/_search_bar.html.erb]: Should this button have fallback text for the image?

If this is an un-styled component, is the image needed at all? I’d love to see a screenshot.

File: source/unstyled.html.erb [hunk 4]
```diff
@@ -70,6 +70,16 @@
     <div class="refill-snippet"><%= code_for("unstyled/footer") %></div>
   </section>
 
+  <section id="search-bar" class="refill">
+   <div class="refills-wrapper">
+     <div class="line-behind-text"><h6>Search bar</h6></div>
```
Comment (tysongach) [inline source/unstyled.html.erb]: `h6` should go on a new line

File: source/stylesheets/refills/unstyled/_search-bar.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
+.search-bar {
+  width: 100%;
+
+  form {  
+    position: relative;
+
+    input[type=search] {
+      @include box-sizing(border-box);
```
Comment (tysongach) [inline source/stylesheets/refills/unstyled/_search-bar.scss]: No need to prefix `box-sizing` anymore.

File: source/unstyled.html.erb (hunk 4 above)
Comment (Magnus-G) [inline source/unstyled.html.erb]: Maybe we can do a PR for all of these after the unstyled elements are all merged?

**Comment by Magnus-G** [comment]
image

File: source/stylesheets/refills/unstyled/_search-bar.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
+.search-bar {
+  width: 100%;
```
Comment (tysongach) [inline source/stylesheets/refills/unstyled/_search-bar.scss]: Why is this needed?

File: source/stylesheets/refills/unstyled/_search-bar.scss [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+  form {  
+    position: relative;
+  }
+
+  input {
+    box-sizing: border-box;
```
Comment (tysongach) [inline source/stylesheets/refills/unstyled/_search-bar.scss]: Why is this needed?

**Comment by tysongach** [comment]
If this is an “unstyled” component, do we need to have an image within the button? Personally, I would immediately strip it out, because:

1. Usually a button with the word “Search” in it will get me pretty far to start with; i.e. no icon
- If I did want an icon in the button, I would use SVG
- Right now, there’s no fallback text for the button image

File: source/stylesheets/refills/unstyled/_search-bar.scss [hunk 8]
```diff
@@ -0,0 +1,26 @@
+.search-bar {
+  width: 100%;
+
+  form {  
+    position: relative;
```
Comment (tysongach) [inline source/stylesheets/refills/unstyled/_search-bar.scss]: Why is this needed?
